run-tests-with-build-optawebs-parent/optawebs-project-sources-test/common.sh

```diff
@@ -28,9 +28,13 @@ function wait_for_url() {
   local _increment=1
 
   local _spent=0
-  while [[ "200" != $(curl -LI "${_application_url}" -o /dev/null -w '%{http_code}' -s) && ${_spent} -lt ${_timeout_seconds} ]]
-  do
-    sleep ${_increment}
+  while [[ "200" != $(curl -LI "${_application_url}" -o /dev/null -w '%{http_code}' -s) ]]; do
+    if [[ "${_spent}" -eq "${_timeout_seconds}" ]]; then
+      echo "Timeout ${_application_url} is not reachable"
+      store_logs_from_pods "target"
+      exit 1
```

Comment:
@dupliaka I think this can be problematic because if the URL doesn't become reachable the script will bail out without doing clean up (`oc delete project ...`). If that is intended and you're OK with the fact that a manual cleanup might be necessary every time this happens, ignore this.